sdk/core/System.Net.ClientModel/src/ModelReaderWriter/ModelReaderWriter.cs

```diff
@@ -12,6 +13,21 @@ namespace System.Net.ClientModel.Core
     /// </summary>
     public static class ModelReaderWriter
     {
+        /// <summary>
+        /// Creates an instance of a model from an instance of an anonymous type.
+        /// </summary>
+        /// <typeparam name="T">The type of the model to create.</typeparam>
+        /// <param name="jsonSerializable">An instance of an anonymous type representing the model</param>
+        /// <returns>A <typeparamref name="T"/> instance.</returns>
+        public static T? Create<T>(object jsonSerializable) where T : IJsonModel<T>
+        {
+            BinaryData json = BinaryData.FromObjectAsJson(jsonSerializable);
+            var reader = new Utf8JsonReader(json);
+            IJsonModel<T> instance = (T)Activator.CreateInstance(typeof(T), true)!;
+            T? read = instance.Read(ref reader, ModelReaderWriterOptions.DefaultJsonOptions);
```

Comment:
Would we want an overload that lets a caller pass their own ModelReaderWriterOptions?  Or would we wait to add it until there's a clear need?  (I ask primarily because all the other methods on this types already provide that overload so there is a bit of a lack of symmetry here with the current approach)

Reply:
I don't think we need this. People who use the wire format would have to understand the wire format, which is not visible in the APIs. The data/json format corresponds 1-1 to the public APIs.